x-pack/plugins/apm/server/routes/alerts/register_transaction_duration_anomaly_alert_type.ts

```diff
@@ -127,6 +131,9 @@ export function registerTransactionDurationAnomalyAlertType({
           return {};
         }
 
+        // Window size must be at least 30, does like this to support rules created before this change where default was 15
+        const windowSize = Math.max(ruleParams.windowSize, 30);
```

Comment:
You would need to take windowUnit into account here too.

Reply:
done: I use datemath to get the diff now https://github.com/elastic/kibana/pull/127973/commits/0794eb545586ce2cd1b94608f53711e49677850b